order/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from order.views import OrderListView 
+
+urlpatterns = [
+    path('/order_list', OrderListView.as_view())
```

Comment:
최종 엔드포인트가 ip:8000/order/order_list 가 되나요?
ip:8000/order 만으로도 충분할 것 같습니다.
```suggestion
    path('', OrderListView.as_view())
```

Reply:
넵 알겠습니다!